RecoEgamma/ElectronIdentification/plugins/VIDUsageExample.cc

```diff
@@ -0,0 +1,123 @@
+/* This module shows how to access the electron ID results using the   
+** VID Framework  
+** It works for both AOD and miniAOD 
+**
+** Author: Sam Harper borrowing liberaly from Ilya Kravchenko and 
+**         Lindsey Gray's examples
+**
+*/
+
+
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "DataFormats/Common/interface/ValueMap.h"
+#include "DataFormats/Common/interface/View.h"
+#include "DataFormats/PatCandidates/interface/Electron.h"
+#include "DataFormats/EgammaCandidates/interface/GsfElectron.h"
+#include "DataFormats/EgammaCandidates/interface/GsfElectronFwd.h"
+
+
+
+class VIDUsageExample : public edm::EDAnalyzer {
```

Comment:
all new code should have thread-aware modules.
Please change to edm::stream:: (seems appropriate here)